components/email/client/editorial_signup.coffee

```diff
@@ -84,27 +85,25 @@ module.exports = class EditorialSignupView extends Backbone.View
       error: ->
         cb null
 
-  showEditorialCTA: ->
-    @test = splitTest('editorial_cta_banner')
-    @outcome = @test.outcome()
-    if @outcome is 'modal' or $('body').hasClass('body-transparent-header')
-      @outcome = 'modal'
+  showEditorialCTA: (outcome) ->
```

Comment:
I wanted to let outcome be an argument here so we could pass in the outcome manually if we wanted to, like for /magazine